Logger/AbstractBaseEntity.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Logger
+{
+    public abstract class AbstractBaseEntity : IEntity
```

Comment:
```suggestion
    public abstract class BaseEntity : IEntity
```
Typically you wouldn't put `abstract` in the class name